src/docfx/build/Build.cs

```diff
@@ -63,7 +63,13 @@ internal static class Build
 
                 bool ShouldBuildFile(Document file)
                 {
-                    return file.ContentType != ContentType.Unknown && fileListBuilder.TryAdd(file);
+                    if (file.ContentType == ContentType.Unknown)
+                        return false;
+                    if (!fileListBuilder.TryAdd(file))
+                        return false;
+                    if (file.ContentType == ContentType.Resource && !file.Docset.Config.Output.CopyResources)
+                        return false;
```

Comment:
Will we build metadata for resource? If so, resource *should* be built when copyResources=true.

Reply:
we don't build resource metadata, it is just a copy, but it is a good point though, putting it here will cause other problems.